packages/client/src/pages/Game/towers/Tower.tsx

```diff
@@ -1,21 +1,41 @@
-import TDEngine, { ITwoDCoordinates } from "../engine/TDEngine";
+import TDEngine, {
+  ITwoDCoordinates,
+  TTowerSpriteTypes,
+} from "../engine/TDEngine";
 import Enemy from "../enemies/Enemy";
-import Projectile, { IProjectile } from "../projectiles/Projectile";
+import Projectile from "../projectiles/Projectile";
+
+export type TTowerParamsDimensions =
+  | "cannonWidth"
+  | "cannonHeight"
+  | "cannonOffsetX"
+  | "cannonOffsetY";
```

Comment:
```suggestion
export type TTowerParamsDimensions =
  | "Width"
  | "Height"
  | "OffsetX"
  | "OffsetY";
```

Reply:
There is a few entities with Width of Height props, it's hard to define what the heck is you are getting, so 

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.